website/pages/deployment/stackinfo.mdx

```diff
@@ -0,0 +1,112 @@
+import {Callout} from "nextra/components"
+
+# Stack Components
+
+The All In One stack provides a number of services which would
+usually be hosted separately, but are combined together here
+for the convenience of users.
+
+Below these are listed, along with a short description of their function,
+and with the local web addresses and user accounts.
+Note that these user accounts are only intended for local training purposes.
+If you deploy this stack for public access then these must be changed.
+
+## TRE-FX Website (Submission Layer)
+
+<Callout>
+**Web Address:** [http://localhost:8888](http://localhost:8888)
```

Comment:
In the rest of the docs we have the Submission Layer UI address as http://localhost:7220 which is the first port option in the docker-compose.yml. I don't mind which we choose but we best to keep it consistent